packages/common/src/store/challenges/selectors/profile-progress.ts

```diff
@@ -42,6 +43,18 @@ export const getHandleExists = (state: CommonState) => {
   return !!curUser.handle
 }
 
+const isValidProfilePicture = (input: User['profile_picture']) => {
+  if (!input) return false
+  const validSizes = [
```

Comment:
nit: should this live outside the function body so that it isn't reallocated each call?